frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -127,6 +127,21 @@ function ReaderHighlight:genHighlightDrawerMenu()
             end,
             callback = function()
                 self.view.highlight.disabled = not self.view.highlight.disabled
+                self.view.highlight.zoom =  self.view.highlight.disabled
+            end,
+            hold_callback = function(touchmenu_instance)
+                self:toggleDefault()
```

Comment:
The new menu item triggers the same `self:toggleDefault()` - should probably be another one, or adapted to decide which of highlight or zoom (or none) should be the default.

Reply:
Done